src/ui/ContentCardConnected/ContentCardConnected.js

```diff
@@ -1,64 +0,0 @@
-import React, { memo } from 'react';
```

Comment:
I mentioned this earlier in the `SearchFeed` stuff, but we're probably going to need to bring this and it's other related files back because of our custom card mappings.

Reply:
I say we just try and customize the new stuff the same way we customized the old stuff

Reply:
You mean customize it in core? I can't think that they'd want any of that.

Reply:
Or bringing over the new files and customizing from there?